### ISSUE TITLE
geof:transform: how can it work in practice?

### ISSUE THREAD

**Issue by FransKnibbe** [opengeospatial/ogc-geosparql#302]
`geof:transform` was added to GeoSPARQL 1.1, but I wonder how it can be of practical use. If a human or machine encounters a SPARQL endpoint that is declared to support the Geometry Extension, how can it be known which CRS IRIs can be used?

**Comment by situx** [comment]
It cannot be known, but the GeoSPARQL 1.0 standard suggests identifiers for CRS as URIs in the opengis namespace. In practice implementers allowed for these URIs to be used to point to e.g. epsg definitions in local databases

**Comment by FransKnibbe** [comment]
It seems to me that geo:transform requires even more CRS knowledge. If data from different datasets are compared, it is possible to match CRSs by just comparing their URIs. So even though details on the CRS are missing, it is still possible to make use of a CRS URI. For coordinate transformation the actual CRS data are needed. So it seems to me that this function can not be used by automated procedures working with Web data.

**Comment by situx** [comment]
Yes, you are correct. However, the situation is no different from e.g. using the geof:sfEquals function.
If the inputs of geof:sfEquals are two literals in two different CRS as identified by their CRS URI, then you also, internally need to convert one literal into the CRS of the other to perform the comparison operation.

**Comment by SimonBin** [comment]
here is an example how to use it:
```sparql
  BIND(geof:transform(?riverGeom, <http://www.opengis.net/def/crs/EPSG/0/5243>) AS ?riverGeomEpsg)
  BIND(if(bound(?riverGeomEpsg), 'success', 'failed') as ?ok)
```
